src/components/dashboard/dashboardViews/ParticipantDashboardView.tsx

```diff
@@ -120,11 +159,16 @@ const ParticipantDashboard: React.FC<Props> = props => {
           ) : (
             branches.map((branch: any) => (
               <Tile
-                key={branch.name}
+                key={branch._id}
                 title={branch.name}
+                branchId = {branch._id}
                 description={`${new Date(branch.settings.open).toDateString()} - ${new Date(
                   branch.settings.close
                 ).toDateString()}`}
+                status={branch._id == currApp?.applicationBranch?._id ? currStatus : "Not Started"}
+                hasApplication = {Object.keys(currApp).length != 0}
+                chooseBranch={chooseBranch}
+                currAppLink={branch._id == currApp?.applicationBranch?._id ? `/${currentHexathon._id}/application/${currApp._id}` : ""}
```

Comment:
Instead of passing in `branch._id` multiple times in different locations, just pass to this component the entire object `branch={branch}`. That way, the component can do what it needs to from the different props. This helps separate responsibility, the the Dashboard component doesn't need to know about all the intricacies.